consensus/checks.go

```diff
@@ -197,9 +197,6 @@ func (consensus *Consensus) onPreparedSanityChecks(
 
 func (consensus *Consensus) viewChangeSanityCheck(msg *msg_pb.Message) bool {
 	consensus.getLogger().Debug().
-		Uint64("blockNum", msg.GetConsensus().BlockNum).
-		Uint64("viewID", msg.GetConsensus().ViewId).
-		Str("msgType", msg.Type.String()).
```

Comment:
you can still use the msg.GetType().String() right?

Reply:
yea, but it's not very necessary. Just to be safe, let's make it same as T3.